test/Tizen.NUI.Devel.Tests.Ubuntu/Tizen.NUI.Devel.Tests/Program.cs

```diff
@@ -82,25 +88,47 @@ protected override void OnCreate()
             };
             root.Add(mainTitle);
 
-            tlog.Debug(tag, "OnCreate() END!");
+            Thread.CurrentThread.Name = "main";
+            trunner = new NUnitLite.TUnit.TRunner();
+            trunner.LoadTestsuite();
 
-            timer = new Timer(1000);
-            timer.Tick += OnTick;
-            timer.Start();
+            asyncThreadMgr = TAsyncThreadMgr.GetInstance();
+            methodExecutionResetEvent = asyncThreadMgr.GetMethodExecutionResetEvent();
+            methodExecutionResetEvent.Reset();
 
+            Task t = Task.Run(() =>
```

Comment:
could be better to be named what task for?

Reply:
Thank you for review, I will change it in next patch.